spec/routing/id_verification_routing_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+describe 'Identity Verification Routes', type: :routing do
+  GET_ROUTES = %w(
+    verify
+    verify/activated
+    verify/address
+    verify/cancel
+    verify/confirmations
+    verify/fail
+    verify/finance
+    verify/finance/other
+    verify/phone
+    verify/review
+    verify/session
+    verify/session/dupe
+    verify/usps
+  ).freeze
+
+  PUT_ROUTES = %w(
+    verify/finance
+    verify/phone
+    verify/review
+    verify/session
+    verify/usps
+  ).freeze
+
+  DELETE_ROUTES = %w(
```

Comment:
for organization, instead of keeping separate verb rountes for each, what if we tracked them as an array of hashes?

```ruby
VERIFY_ROUTES = [
  { put: 'verify/finance' },
  { delete: 'verify/session' },
]
```